src/config/configDefaults.ts

```diff
@@ -1,10 +1,15 @@
-export const SHOW_MUTATION_ASSESSOR = true;
+export const SHOW_MUTATION_ASSESSOR = false;
 
-export const ANNOTATION_QUERY_FIELDS = [
+export function annotationQueryFields() {
+    let fields = DEFAULT_ANNOTATION_QUERY_FIELDS;
```

Comment:
I think this can be const
```suggestion
    const fields = DEFAULT_ANNOTATION_QUERY_FIELDS;
```